support-frontend/assets/pages/supporter-plus-thank-you/components/thankYouHeader/heading.tsx

```diff
@@ -183,6 +186,7 @@ type HeadingProps = {
 	amount: number | undefined;
 	currency: IsoCurrency;
 	contributionType: ContributionType;
+	ratePlanKey?: string;
```

Comment:
Do we have a stricter type for `ratePlanKey` rather than a string, so we can ensure it's valid? It looks like `ActiveProductKey` fills this purpose?